<launch>
  <!-- Arguments -->
  <arg name="fake_move_base" default="true"/>
  
  <include file="$(find turtlebro_delivery)/launch/deps.launch">
    <arg name="fake_move_base" value="$(arg fake_move_base)"/>
  </include>  

  <!-- Run Aruco service -->
  <node pkg="turtlebro_delivery" type="delivery.py" name="delivery_node" output="screen" required="true" />
</launch>
